mofiebiger/ember-trivia-game
# Add .circleci/config.yml

# ✨ What Changed & Why

Configuring CircleCI project setup.

## 👹 Before

No visual changes

## 😇 After

No visual changes

## 🌈 Testing

1. Observe .circleci/config.yml file is added
1. Observe CircleCI tests are passing when commits are made
